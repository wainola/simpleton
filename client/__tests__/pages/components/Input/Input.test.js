const { Input } = require('../../../../pages/components/Input');
const { mount } = require('enzyme');

describe('<Input />', () => {
  it('should render without errors', () => {
    const component = mount(<Input type="text" placeholder="ingrese su nombre" title="Nombre" />);

    expect(component).toMatchSnapshot();
  });

  it('should have an span element and an input element', () => {
    const component = mount(
      <Input type="text" placeholder="ingrese su nombre" title="Nombre del usuario" />
    );

    expect(component.find('span')).toHaveLength(1);
    expect(component.find('input')).toHaveLength(1);
  });
});
