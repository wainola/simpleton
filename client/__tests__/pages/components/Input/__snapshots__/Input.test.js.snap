// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Input /> should render without errors 1`] = `
<Input
  placeholder="ingrese su nombre"
  title="Nombre"
  type="text"
>
  <span
    className="input-title"
  >
    Nombre
  </span>
  <input
    placeholder="ingrese su nombre"
    type="text"
  />
</Input>
`;
